test/cache.test.js

```diff
@@ -9,12 +9,14 @@ const fastify = require('fastify')
 test('cache property gets added to instance', (t) => {
   t.plan(2)
   const instance = fastify()
-  instance.register(plugin, (err) => {
-    if (err) t.threw(err)
-
-    t.ok(instance.cache)
-    t.ok(instance.cache.set)
-  })
+  instance
+    .register(plugin, (err) => {
```

Comment:
This callback can be removed. The `register` method doesn't invoke one any longer.